tox.ini

```diff
@@ -20,10 +20,26 @@ commands = nosetests --ignore-files=test_appengine\.py {posargs}
 [testenv:cover]
 basepython = python2.7
 commands =
-    nosetests --with-xunit --with-xcoverage --cover-package=oauth2client --nocapture --cover-erase --cover-tests --cover-branches --ignore-files=test_appengine\.py --cover-min-percentage=60
+    nosetests \
```

Comment:
You can if you like. When #212 is closed we'll make it fail under 100, but it can't fail on coveralls, so it's a bit delicate.

For `gcloud-python` I ended up just wrapping all the flags in a bash script and then calling it with the cover-min flag in the `cover` rule but not in the `coveralls` rule: https://github.com/GoogleCloudPlatform/gcloud-python/blob/ac13419365b254b2fbd6fb40eba558859df8341f/tox.ini#L15-L33